src/values.js

```diff
@@ -169,8 +169,26 @@ export class DurationValue extends IntegerValue {
   toJS(value) {
     let quantity, unit;
 
+    const millisecondsUnits = ['ms', 'millisecond', 'milliseconds'];
+    const secondsUnits = ['s', 'sec', 'secs', 'second', 'seconds'];
+    const minutesUnits = ['m', 'min', 'mins', 'minute', 'minutes'];
+    const hoursUnits = ['h', 'hr', 'hrs', 'hour', 'hours'];
+    const daysUnits = ['d', 'day', 'days'];
+
+    const unitsPatterns = [
+      ...millisecondsUnits,
+      ...secondsUnits,
+      ...minutesUnits,
+      ...hoursUnits,
+      ...daysUnits,
+    ]
+      .reduce((reduced, current) => `${reduced}|${current}`, '')
+      .slice(1);
```

Comment:
If you leave off the second parameter in `.reduce()`, it defaults to the first item in the array, so you could skip the call to `.slice(1)`.

Reply:
this is reducing the array to a string which looks something like:
```
'|ms|millisecond|milliseconds...
```
the `.slice(1)` slices off the leading `|`.

Reply:
Yeah, and if you leave off the `''` in the call to reduce, it will process `'ms'` first, so it won't have the extra leading `|`.

Reply:
ah! gotcha! TIL

Reply:
```js-shell
>>> ['a', 'b', 'c'].reduce((s, x) => `${s}|${x}`, '')
"|a|b|c"
>>> ['a', 'b', 'c'].reduce((s, x) => `${s}|${x}`)
"a|b|c" 
```

Reply:
yup. fixed!